charactersheet/charactersheet/viewmodels/dm/encounter_sections/environment_section.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+function EnvironmentSectionViewModel(parentEncounter) {
+    var self = this;
+
+    self.template = 'environment_section.tmpl';
+    self.encounterId = parentEncounter.encounterId;
+
+    self.name = ko.observable();
+    self.visible = ko.observable(false);
+
+    self.imageUrl = ko.observable();
+    self.weather = ko.observable();
+    self.terrain = ko.observable();
+    self.description = ko.observable('');
+
+    self.previewTabStatus = ko.observable('active');
+    self.editTabStatus = ko.observable('');
+
+    //Public Methods
+
+    self.init = function() {
+        Notifications.global.save.add(self.save);
+        Notifications.encounters.changed.add(self._dataHasChanged);
+   };
+
+    self.load = function() {
```

Comment:
It creates one at save time. No need to make it here since that's not "loading" anything.